.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                python-version: ['3.7', '3.9', '3.10', '3.11', '3.12']
+                python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
The test for 3.8 seems to be failing still (#228). Probably related to #279, so maybe leave it disabled until we figure it out.